libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/LUIS/LuisAdaptiveRecognizer.cs

```diff
@@ -81,10 +81,13 @@ public LuisAdaptiveRecognizer()
         public HttpClientHandler HttpClient { get; set; }
 
         /// <summary>
-        /// Gets or sets a value indicating whether to log personal information that came from the user to telemetry.
+        /// Gets or sets the flag to determine if personal information should be logged in telemetry.
         /// </summary>
-        /// <value>If true, personal information is logged to Telemetry; otherwise the properties will be filtered.</value>
-        public bool LogPersonalInformation { get; set; } = false;
+        /// <value>
+        /// The flag to indicate in personal information should be logged in telemetry.
+        /// </value>
+        [JsonProperty("logPersonalInformation")]
+        public BoolExpression LogPersonalInformation { get; set; } = "=settings.telemetry.logPersonalInformation";
```

Comment:
For all these, we need an update to the LuisRecognizer component schema as well as re-gen the sdk.schema.